tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -411,6 +411,9 @@ def test_fsdp_load_old_checkpoint(
     s3_read_only_prefix: str,
     composer_version: str,
 ):
+    if composer_version == '0.18.1' and state_dict_type == 'full' and precision == 'amp_bf16' and sharding_strategy == 'FULL_SHARD':
+        pytest.skip('TODO: This checkpoint is missing')
```

Comment:
Is this TODO for this PR or subsequent PR?

Reply:
subsequent PR